helpers.lua

```diff
@@ -5,7 +5,7 @@
 
 --]]
 
-local spawn = require("awful.spawn")
+local spawn      = require("awful.spawn")
```

Comment:
This is a good commit. If you separate it from this PR, I can merge it.

Reply:
Just the change to line 8 or you mean all the changes to the file?

Reply:
All changes to the file.